packages/kbn-legacy-logging/src/log_format.ts

```diff
@@ -56,15 +56,15 @@ export abstract class BaseLogFormat extends Stream.Transform {
 
   abstract format(data: Record<string, any>): string;
 
-  filter(data: Record<string, unknown>) {
+  filterData(data: Record<string, unknown>) {
```

Comment:
Note to reviewers: I had to rename this function as it's conflicting with the upstream (no pun intended) function on the `Transform` class [by the same name](https://nodejs.org/api/stream.html#readablefilterfn-options). The new types installed in this PR suddenly highlighted that